gtest_parallel.py

```diff
@@ -757,14 +757,16 @@ def find_tests(binaries, additional_args, options, times):
     command = [test_binary] + additional_args
     if options.gtest_also_run_disabled_tests:
       command += ['--gtest_also_run_disabled_tests']
+    if options.size != '':
+      command += ['--size=' + options.size]
```

Comment:
Does it need the `*`?

Reply:
If `--size=*` isn't passed as a command-line option to this script, it defaults to Small tests only. 

So, these lines are saying "if a `--size` argument was provided, make sure the executables use it, otherwise don't pass a `--size` argument at all"